### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Phionamary**
This PR delivers the User Interface for the Fast-Food-Fast application

### CODE REVIEW

**Comment by KicaRonaldOkello** [comment]
Please use [Delivers #______] in your commits once you have finished a particular story. It prepares the story to be rejected or accepted

File: UI/CSS/orders.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
+@media only screen and (max-width: 640px) {
```
Comment (kdaglas) [inline UI/CSS/orders.css:1]: Are you doing responsiveness for only those screens or is there more... forexample those small iphone screens...
any way responsiveness exceeds expectation and it good you are exceeding expectations...

**Comment by joelethan** [comment]
Phiona have ue hosted with gh-pages, if have, you put the link in the description so one can easily look at ur design

**Comment by kdaglas** [comment]
am with @KicaRonaldOkello... let your pivotal stories be in sync with the github work flow

File: UI/CSS/orders.css (hunk 1 above)
Comment (Phionamary) [reply UI/CSS/orders.css:1]: I am planning to add in more screen sizes with time....

> Phiona have ue hosted with gh-pages, if have, you put the link in the description so one can easily look at ur design

**Comment by Phionamary** [comment]
@KicaRonaldOkello  and @kdaglas My github and pivotal tracker are in sync... What i do on github triggers pivotal...

File: UI/admin.html [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalabe=no">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge, chrome=1">
+        <meta name="HandheldFriendly" content="true">
+        <link rel="stylesheet" type="text/css" href="../UI/CSS/admin.css">
+        <script type="text/javascript" src="js/scripts.js"></script>
+    </head>
+
+    <body>
+
+        <style>
```
Comment (66hakeem) [inline UI/admin.html:18]: You could put this in your css file and define an id for the body tag to do away with the inline css.

**Pull request merged**